src/components/container/ConnectEhr/ConnectEhr.stories.tsx

```diff
@@ -52,32 +52,32 @@ export const NotEnabledDisplayError = {
 };
 
 export const Enabled = {
-    args: { previewState: "enabled", applicationUrl: "preview" },
+    args: { previewState: "enabled", onClick: () => { console.log("PREVIEW: Opening the connect EHR application.") } },
```

Comment:
Minor:  Could probably define a single const function and just pass it to all these.